### DISCUSSION TITLE
no `no-unused-css` replacement

### DISCUSSION THREAD

**Discussion by JounQin** [angular-eslint/angular-eslint#2299] [New Rule Proposals]
**Description and reproduction of the issue**

http://codelyzer.com/rules/no-unused-css/

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint       `        | `X.Y.Z` |
| `TypeScript`                       | `X.Y.Z` |
| `ESLint`                           | `X.Y.Z` |
| `node`                             | `X.Y.Z` |

```sh
# Please run `npx ng version` in your project and paste the full output here:


```

- [ ] I have tried restarting my IDE and the issue persists.
- [ ] I have updated to the latest supported version of the packages and checked my `ng version` output per the instructions given here.

**Comment by JamesHenry** [comment]
@JounQin Yes this is documented on the README that there is no equivalent yet and that PRs are welcome at any time from folks who use the rule. I'm happy to leave this open though
